app/code/Magento/Store/Ui/DataProvider/WebsiteDataProvider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Store\Ui\DataProvider;
+
+use Magento\Store\Model\ResourceModel\Website\Grid\CollectionFactory as WebsiteCollectionFactory;
+use Magento\Store\Model\ResourceModel\Website\Grid\Collection as WebsiteCollection;
+
+/**
+ * Class WebsiteDataProvider
+ */
+class WebsiteDataProvider extends \Magento\Ui\DataProvider\AbstractDataProvider
+{
+    /**
+     * @var WebsiteCollection
+     */
+    protected $collection;
```

Comment:
Please change the type to `private`

Reply:
It is unnecessary here, I will remove this. 
`$collection` is declared in parent class.

Reply:
@slavvka removed.